example/android/app/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.chirp_flutter_example">
```

Comment:
Is this an auto-generated Manifest? What is the purpose of this file?

Reply:
Yeah autogenerated, not sure exactly but there are 3 generated folders - `debug`, `main` and `profile`.

Reply:
Well. we definitely need to rename the package on all 3 files (or re-generate them).
Not sure about the permissions, do we need to include them here as well?